### ISSUE TITLE
I can click on messages through modal

### ISSUE THREAD

**Issue by simonricoo** [status-im/status-desktop#1829]
https://user-images.githubusercontent.com/52249767/107641002-594a1c00-6c73-11eb-8750-898ebe6403ac.mov

**Comment by PascalPrecht** [comment]
Unfortunately, I'm not able to reproduce this one.

**Comment by jrainville** [comment]
I think this has been fixed by Richard and/or me while fixing the compact mode hovers

**Comment by PascalPrecht** [comment]
Closing as resolved @simonricoo let us know if you run into this again.

**Comment by simonricoo** [comment]
I ran into this again: 

https://user-images.githubusercontent.com/52249767/109279867-90dfc900-781a-11eb-8cc9-913cc5a67b43.mov

**Comment by PascalPrecht** [comment]
What version are you on. I'm on latest `master` and still can't reproduce it..

**Comment by PascalPrecht** [comment]
Updating here:

This is reproducible in compact mode, which is now the default (hence, why I couldn't reproduce this before).
I've debugged this and clearly what happens is that the message that are being clicked through the modal receive a `isMessageActive = true` state.

This causes the message change its color and render the context menu buttons for emojis etc.

Now, the interesting part is, is that all the handlers that potentially could set `isMessageActive` are **not** executed. At least not that I could see (I put log statements in them to verify and didn't get any when clicking messages while popup is open).
In addition, I've commented out **all** lines of code that actually set `isMessageActive` to `true` and they **still** change their active state when clicked while the popup is open.

Also notice that, multiple messages can be made active when the popup is open, which is not possible when it's closed. I fear that this is a bug in Qt/QML, not really sure what else to do at this point.
